pex/BUILD

```diff
@@ -22,7 +22,7 @@ genrule(
         # Workaround really long shebang lines breaking on linux:
         # Use a /tmp path, but keep the actual venv inside the bazel outdir.
         # Avoids having to worry about cleanup, even if sandboxing is off.
-        TMPF=$$(mktemp)
+        TMPF=$$(mktemp 2>/dev/null || mktemp -t tmp)
```

Comment:
@benley yeah this was a weird one, it appears the behavior was fixed to match gnu in OS X 10.11, but older versions of OS X require the `-t` flag. Only encountered it due to a coworker running into this issue.

https://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x